bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckablePullRequest.java

```diff
@@ -155,7 +156,7 @@ static List<Review> filterActiveReviews(List<Review> allReviews, String targetRe
         return List.copyOf(reviewPerUser.values());
     }
 
-    Hash commit(Hash finalHead, Namespace namespace, String censusDomain, String sponsorId, Hash original) throws IOException, CommitFailure {
+    Hash commit(Hash finalHead, Namespace namespace, String censusDomain, String sponsorId, Hash original, List<Comment> comments) throws IOException, CommitFailure {
```

Comment:
Can you also replace the usage in `commitMessage` in this class.

Reply:
Sure